src/mad_spark_alt/core/robust_orchestrator.py

```
@@ -0,0 +1,281 @@
+        successful_registrations = 0
+        for agent_class in [
+            QuestioningAgent,
+            AbductionAgent,
+            DeductionAgent,
+            InductionAgent,
+        ]:
+            try:
+                self.registry.base_registry.register(agent_class)  # type: ignore[type-abstract]
+                successful_registrations += 1
+                logger.debug(f"Successfully registered template agent: {agent_class.__name__}")
+            except Exception as e:
```

Ruff (0.11.9)

Do not catch blind exception: `Exception`

(BLE001)